hack/generator/pkg/codegen/testdata/ARMCodeGeneratorPipeline.golden

```diff
@@ -0,0 +1,46 @@
+Expected Pipeline Stages for ARM Code Generation
+------------------------------------------------
+loadSchema                                     Load and walk schema
```

Comment:
Hmm, this file being added seems wrong? I think it exists already but maybe with a different name?

Reply:
Odd, it's definitely surplus. Will remove.